site/cds_rdm/inspire_harvester/jobs.py

```diff
@@ -7,19 +7,17 @@
 
 """Jobs module."""
 
-from datetime import timezone
+from datetime import datetime
 
 from invenio_i18n import gettext as _
 from invenio_vocabularies.jobs import ProcessDataStreamJob
 from marshmallow import Schema, ValidationError, fields, validates_schema
-from marshmallow_utils.fields import TZDateTime
 
 
 class InspireArgsSchema(Schema):
     """Schema of task input arguments."""
 
-    since = TZDateTime(
-        timezone=timezone.utc,
+    since = fields.Date(
```

Comment:
this field should be inherited from parent class (there is a generic schema implementing `since` field, and it should be used.
are you sure timezone is not needed?

Reply:
I had to redefine the field to be able to change the description (the one in the parent class has "YYYY-MM-DD HH:mm" format). 
Because in INSPIRE we can not search by datetime and only by date, I don't see the reason to support timezone